docker/lerobot-compose/Dockerfile

```diff
@@ -0,0 +1,79 @@
+FROM nvidia/cuda:12.2.2-devel-ubuntu22.04
```

Comment:
Most of this build is identical to `docker/lerobot-gpu-dev`, except I added the poetry install step and `POETRY_EXTRAS` env variable.